packages/gatsby-link/.babelrc

```diff
@@ -1 +1,8 @@
-{}
+{
```

Comment:
This file change is not necessary since we use `microbundle` to generate the output.

You can check out https://github.com/gatsbyjs/gatsby/tree/master/packages/gatsby-script to see how it's used (also has relevant `package.json` bits)